driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1408,7 +1408,7 @@ protected Builder withScyllaCloudConnectionConfig(ScyllaCloudConnectionConfig co
                     new ScyllaCloudSniEndPointFactory(
                         proxyAddress, currentDatacenter.getNodeDomain()))
                 .withSSL(
-                    (config.getCurrentAuthInfo().isInsecureSkipTlsVerify()
+                    (config.getCurrentDatacenter().isInsecureSkipTlsVerify()
                         ? config.createBundle().getInsecureSSLOptions()
                         : config.createBundle().getSSLOptions()))
```

Comment:
A potential problem I see (and I didn't expect at the beginning): Previously, there was only a single `insecureSkipTlsVerify`, so we could set it here with `withSSL` on a session. But now, you could have different configurations of `insecureSkipTlsVerify` per each datacenter - but here `currentDatacenter` determines the configuration for all SSL connections.